container/openejb-core/src/main/java/org/apache/openejb/assembler/classic/ImportSql.java

```diff
@@ -35,6 +35,13 @@
 import java.util.Enumeration;
 import java.util.Locale;
 
+/**
+ * This class should no longer be used to import database objects. 
```

Comment:
OK.

Reply:
Would you like me, to put a link to the Jira web page, where the TOMEE-1380 task is located ?

Reply:
I was thinking about something like: "See TOMEE-1380 for details." - but this is optional. If you think it could be helpful, add it, if not don't. It's up to you :-)

Reply:
Ok